src/test/java/io/in_toto/lib/AppTest.java

```diff
@@ -1 +0,0 @@
-package io.in_toto.lib;
```

Comment:
Is this not needed? or?

Reply:
It's [here](https://github.com/in-toto/in-toto-java/blob/f7d950fbb89cda415b6206765a8c4b37bd6244f4/src/test/java/io/github/in_toto/lib/AppTest.java#L1) now

Reply:
No, it's not. I just modified the same file,
and changed the single line of code from "package io.in_toto.lib;" to "package io.github.in_toto.lib;"
and changed the path from "src/test/java/io/in_toto/lib/AppTest.java" to "src/test/java/io/github/in_toto/lib/AppTest.java"

But, the git diff detected this as new file and considered the old file as deleted.